firebase-vscode/src/utils/settings.ts

```diff
@@ -7,17 +7,23 @@ export interface Settings {
   readonly shouldShowIdxMetricNotice: boolean;
 }
 
-const FIREBASE_BINARY =
-  // Allow defaults via env var. Useful when starting VS Code from command line or Monospace.
-  process.env.FIREBASE_BINARY ||
-  // TODO: Temporary fallback for bashing, this should probably point to the global firebase binary on the system
-  "npx -y firebase-tools@latest";
+// TODO: Temporary fallback for bashing, this should probably point to the global firebase binary on the system
+const DEFAULT_FIREBASE_BINARY = "npx -y firebase-tools@latest";
 
 export function getSettings(): Settings {
   const config = workspace.getConfiguration("firebase");
 
+  // TODO: Consider moving side effect out of getSettings
+  // Persist env var as path setting when path setting doesn't exist
+  if (process.env.FIREBASE_BINARY && !config.get<string>("firebasePath")) {
```

Comment:
According to http://b/373897889 we should check and set user preference (not per-workspace preference)

Reply:
Setting the setting type to `scope: application` should make it _only_ a user setting. I can add ConfigurationTarget.Global to ensure it?

Reply:
`ConfigurationTarget.Global` sounds good, and we need this because we want to keep it manually overridable per workspace if developers really want to (see the other comment).